events/Lambda/CreateEventSourceMapping20150331.json

```diff
@@ -9,6 +9,39 @@
     "mitreAttackTechniques": [
         "T1098 - Account Manipulation"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1059 - Command and Scripting Interpreter",
+            "reason": "The Lambda function might execute code based on the event source data, potentially running JavaScript if included in the payload."
+        },
+        {
+            "technique": "T1071 - Application Layer Protocol",
+            "reason": "Lambda functions can communicate over web protocols, enabling command and control through event source triggers."
+        },
+        {
+            "technique": "T1546 - Event Triggered Execution",
+            "reason": "Event source mappings can be used to trigger Lambda functions, executing code in response to specific events or data."
+        },
+        {
+            "technique": "T1070 - Indicator Removal",
+            "reason": "Adversaries could configure Lambda functions to delete logs or other indicators of their activity upon execution."
+        },
+        {
+            "technique": "T1106 - Native API",
+            "reason": "Utilizing the CreateEventSourceMapping API, adversaries can execute Lambda functions by mapping event sources to them."
```

Comment:
This technique is only for OS application programming interface (API)

Reply:
Removed.